contrib/bash-completion

```diff
@@ -237,6 +237,11 @@ __bob_jenkins()
    __bob_subcommands "add export ls prune push rm set-url set-options" "jenkins"
 }
 
+__bob_project()
+{
+   __bob_complete_path "-d --dev -b --build --destination --sandbox --no-sandbox"
```

Comment:
These options do not fit anymore. Also, the command first has to complete the "generator" and then a "package". For that we'll need a "bob project --list" to get all available generators.